openapi/openapi/src/main/scala/endpoints4s/openapi/model/OpenApi.scala

```diff
@@ -70,7 +70,12 @@ object OpenApi {
 
   private def mapJson[A](map: collection.Map[String, A])(f: A => ujson.Value): ujson.Obj = {
     val result = ujson.Obj()
-    map.foreach { case (k, v) => result.value.put(k, f(v)) }
+    val stableMap = map match {
+      case map: collection.mutable.LinkedHashMap[String, A] => map
+      case map: collection.immutable.ListMap[String, A]     => map
+      case map                                              => map.toSeq.sortBy(_._1)
+    }
+    stableMap.foreach { case (k, v) => result.value.put(k, f(v)) }
```

Comment:
Would you mind adding a brief comment in the code explaining what it does?

Also, I think it could be simplified to:

```suggestion
    val stableMap = map match {
      case seqMap: collection.SeqMap[String, A] => seqMap
      case map                                  => map.toSeq.sortBy(_._1)
    }
    stableMap.foreach { case (k, v) => result.value.put(k, f(v)) }
```

Reply:
makes sense

Reply:
oops SeqMap is not part of 2.12